lib/dry/system/stub.rb

```diff
@@ -0,0 +1,27 @@
+module Dry
+  module System
+    class Container
+      # Incuded only in the Test environment
+      # Sending the message enable_stubs! allow you to stub components after
+      # finalize your container in your tests.
+      #
+      # @api private
+      module Stub
```

Comment:
Feels to me like this could be called `Stubs` rather than just `Stub`? That feels more natural to me since it's about a stubbing _capability_ in general, not just one stub )